### PR TITLE
Release/0.31

### PR DESCRIPTION
**Pull request by wmww**
Did I do everything right?

### CODE REVIEW

**Comment by wmww** [comment]
Hold on, let me rebase and do things in the right order so we don't get conflicts.

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,26 @@
+mir (0.31.2) UNRELEASED; urgency=medium
+
+  * New upstream release 0.31.1(https://github.com/orgs/MirServer/projects/4)
```
Comment (mariogrip) [inline debian/changelog]: this says 0.31.1 and its the link is 404 (maybe its a private project?)

Comment (wmww) [reply debian/changelog]: Whoops, I just copied it. Should I remove that line or change it to something? What does it even mean?

Comment (Saviq) [reply debian/changelog]: Remove the link, we don't have projects for bugfix releases: https://github.com/MirServer/mir/projects

**Comment by AlanGriffiths** [comment]
[Test plan](https://wiki.ubuntu.com/Process/Merges/TestPlans/Mir) completed without problems for 16.04LTS

**Comment by AlanGriffiths** [comment]
[Test plan](https://wiki.ubuntu.com/Process/Merges/TestPlans/Mir) completed without problems for 17.10

**Pull request merged**